MusicTaste/MusicTaste/View Controllers/Login/ConnectViewController.m

```diff
@@ -0,0 +1,61 @@
+#import "ConnectViewController.h"
+#import <Parse/Parse.h>
+#import "ConnectView.h"
+#import "SpotifyAPIManager.h"
+#import "dispatch/dispatch.h"
+#import "MatchmakingViewController.h"
+
+@interface UIViewController ()
+
+@end
+
+@implementation ConnectViewController
+
+#pragma mark - Authorization example
```

Comment:
Is this pragma accurate?

By the way, pragmas are just there to group the code. So you can collapse/annotate sections of your code. But they are not necessary, so if you're not getting value from them just remove them. It's personal preference; I pretty much always remove them, but some people like them. So depends on you. But if you do have pragmas they should accurately reflect the code contained within